tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -567,57 +567,67 @@ def config(self, config, js_config):
 class TestEnableOAuth2RedirectErrorMode:
     def test_scope_error(self, js_config):
```

Comment:
I know these were here before, but these two tests don't really seem to test much different from each other?

The first one could cover both by calling with `sentinel.error_details`, `sentinel.is_scope_invalid` etc. Maybe have the other to test the defaults?

Reply:
Parametrized this